Composer/packages/client/src/pages/botProject/PublishTargets.tsx

```diff
@@ -103,6 +107,22 @@ export const PublishTargets: React.FC<PublishTargetsProps> = (props) => {
     }
   };
 
+  const addNewButton = (
+    <ActionButton
+      data-testid={'addNewPublishProfile'}
+      styles={actionButton}
+      onClick={() => {
+        if (isShowAuthDialog(true)) {
```

Comment:
Will fix. Thanks!

Reply:
This is actually a little more involved (since the "else" clause leads to a different action), but I'm taking this opportunity to clean up some logic and readability issues.